helpers/template.py

```diff
@@ -215,6 +215,8 @@ def _get_value(property_, true_value="", false_value="#",
             "REDIS_MAIN_PORT": config.get("redis_main_port", "6739"),
             "REDIS_CACHE_PORT": config.get("redis_cache_port", "6380"),
             "USE_BACKUP": "" if config.get("use_backup") == Config.TRUE else "#",
+            "USE_WAL_E_BACKUP": "" if config.get("use_wal_e") == Config.TRUE else "#",
```

Comment:
Would you mind to use `_get_value` instead? 
I've noticed that several lines should be updated too ;-( We should open a separate issue for that.

Reply:
No problem! Should I update all instances of `config.get()` with `_get_value()`?